blog/urls.py

```diff
@@ -1,8 +1,10 @@
 from django.conf.urls import url
+from rss_feed.forms import BlogsFeed
```

Comment:
Minor: why not add this class to blogs/views.py?

Reply:
That's probably I'll move this there.